Playbooks/playbook-Dedup_-_Generic.yml

```diff
@@ -346,6 +370,12 @@ inputs:
     simple: "true"
   required: false
   description: If duplicate found - close the incident as duplicate (true\false)
+- key: Time field
```

Comment:
i think its best practise not to have white spaces in inputs, should be `TimeField`